config/v3_2_experimental/schema/ignition.json

```diff
@@ -232,6 +238,70 @@
               "devices"
           ]
         },
+        "luks": {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "label": {
+              "type": ["string", "null"]
+            },
+            "uuid": {
+              "type": ["string", "null"]
+            },
+            "device": {
+              "type": "string"
+            },
+            "cipher": {
+              "type": ["string", "null"]
+            },
+            "keyFile": {
+              "type": ["string", "null"]
+            },
+            "hash": {
```

Comment:
We mean "hash algorithm" here, which is somewhat confusing because elsewhere in the spec we use `hash` to mean the actual bits of a hash.  Thoughts?